packages/api/pages/api/languages/index.ts

```diff
@@ -21,6 +22,7 @@ export default createRoute<void>()
   })
   .post<PostLanguageRequestBody, void>({
     schema: languageSchema,
+    authorize: authorize({ action: 'create', subject: 'User' }),
```

Comment:
```suggestion
    authorize: authorize({ action: 'create', subject: 'Language' }),
```
If I'm getting the casl stuff right, the this should be `Language`. Maybe I'm misunderstanding something.